kornia/utils/image.py

```diff
@@ -65,23 +64,25 @@ def _to_bchw(tensor: torch.Tensor) -> torch.Tensor:
     if len(tensor.shape) == 3:
         tensor = tensor.unsqueeze(0)
 
+    if len(tensor.shape) > 4:
+        tensor = tensor.view(-1, tensor.shape[-3], tensor.shape[-2], tensor.shape[-1])
+
     return tensor
 
 
 def _to_bcdhw(tensor: torch.Tensor) -> torch.Tensor:
```

Comment:
```suggestion
def tensor_to_bcdhw(tensor: torch.Tensor) -> torch.Tensor:
```
@shijianjian @ducha-aiki @lferraz to improve functional naming ?